client/reader/stream/_style.scss

```diff
@@ -595,14 +595,18 @@ $reader-related-card-v2-breakpoint-small: "( max-width: 535px )";
 		}
 
 		.reader-related-card-v2__post {
-			max-height: 220px;
+			max-height: 200px;
 
 			@media #{$reader-related-card-v2-breakpoint-small} {
-				max-height: 170px;
+				max-height: 150px;
 			}
 
 			@media #{$reader-related-card-v2-breakpoint-medium} {
-				max-height: 170px;
+				max-height: 150px;
+			}
+
+			.reader-related-card-v2__excerpt {
+				-webkit-line-clamp: 3;
```

Comment:
I've removed this in: https://github.com/Automattic/wp-calypso/pull/9962/commits/9ca83a422d3898e1ac1a9b154ece848e79b0c4c1. It wasn't actually needed since we're using `max-height`.